dyndns.sh

```diff
@@ -13,12 +13,13 @@ test -z $DOMAIN && die "DOMAIN not set!"
 test -z $NAME && die "NAME not set!"
 
 dns_list="$api_host/domains/$DOMAIN/records"
+dns_list_long="$api_host/domains/$DOMAIN/records?per_page=200"
```

Comment:
@amatashkin thank you for your contribution!

but below code loops indefinitely with a sleep interval which doesn't make sense with your change. I'd suggest a for-loop inside the while loop to check against all pages and if not found, wait for the next cycle.

Reply:
The changes mentioned do not work because the url **$dns_list** is also used for the **PUT** operation. Instead, append _**?per_page=200**_ in the curl command itself like so: https://github.com/zzeneg/digitalocean-dyndns/commit/e7ededa9dfb2fe2c1224f4359ab8a655aad47d88